src/timings.cpp

```diff
@@ -169,14 +169,19 @@ f64 time_stamp(TimeStamp const &ts, u64 freq, TimingUnit unit) {
 	}
 }
 
-void timings_print_all(Timings *t, TimingUnit unit = TimingUnit_Millisecond) {
+void timings_print_all(Timings *t, TimingUnit unit = TimingUnit_Millisecond, bool timings_are_finalized = false) {
```

Comment:
Is there anything that sets `timings_are_finalized` to `true` at any point?

Reply:
Yes, `main.cpp:show_timings():1693` originally called this procedure and then finalized the timings. As I'm iterating over the timings again for the export, I don't want to stop the clock twice. So now it's called first for the stdout output as before, only with the implicit `false`, and then the export calling it sets it to `true`.

Reply:
Okay. Makes sense!